qa/tasks/cephfs/test_misc.py

```diff
@@ -504,6 +504,28 @@ def test_dump_inmemory_log_on_missed_internal_heartbeats(self):
                 return
         self.assertTrue(False, "Failed to dump in-memory logs during missed internal heartbeat")
 
+    def test_session_ls(self):
+        info = self.fs.mds_asok(['session', 'ls'])
+        self.assertIsNotNone(info)
+        mount_a_client_id = self.mount_a.get_global_id()
+        info = self.fs.mds_asok(['session', 'ls', f"id={mount_a_client_id}"])
+        self.assertEqual(len(info), 1)
+        self.assertEqual(info[0]["id"], mount_a_client_id)
+        info = self.fs.mds_asok(['session', 'ls', '--cap_dump'])
+        for i in range(0, len(info)):
+            self.assertIn("caps", info[i])
+
+    def test_client_ls(self):
+        info = self.fs.mds_asok(['client', 'ls'])
+        self.assertIsNotNone(info)
+        mount_a_client_id = self.mount_a.get_global_id()
+        info = self.fs.mds_asok(['client', 'ls', f"id={mount_a_client_id}"])
+        self.assertEqual(len(info), 1)
+        self.assertEqual(info[0]["id"], mount_a_client_id)
+        info = self.fs.mds_asok(['client', 'ls', '--cap_dump'])
+        for i in range(0, len(info)):
+            self.assertIn("caps", info[i])        
```

Comment:
both the test cases are almost the same, either you can include both the cases under the same function or create a helper function like

```
def test_ls(self, cmd: str):
        info = self.fs.mds_asok(cmd)
        self.assertIsNotNone(info)
        mount_a_client_id = self.mount_a.get_global_id()
        info = self.fs.mds_asok(cmd + f" {mount_a_client_id}"))
        self.assertEqual(len(info), 1)
        self.assertEqual(info[0]["id"], mount_a_client_id)
        info = self.fs.mds_asok(cmd + " --cap_dump")
        for i in range(0, len(info)):
            self.assertIn("caps", info[i])
```
and then call it in `test_session_ls` and `test_client_ls`?